tests/operator/operator.go

```diff
@@ -1224,12 +1224,9 @@ spec:
 					Expect(err).ToNot(HaveOccurred(), stderr)
 				}
 
-				// Use Current virtctl to start VM
```

Comment:
I don't fully understand why operator.go tests should depend on virtctl; the operator must work correctly regardless of the mechanism that started the VM. However, since the original author went into the trouble of stating that this is intentional, I think that the commit message and the PR text should try to understand and explain the issue.

It may be an expensive way to check that a new `virtctl` can talk to an old kubevirt.

Reply:
I don't see how it makes any difference using `virtClient` instead of virtctl in this regard .

The lines you see here are literally identical to what virtctl does today:

https://github.com/kubevirt/kubevirt/blob/a3a9e83da6fff2de37b7fd87efb43466cd6e07d4/pkg/virtctl/vm/start.go#L59

And even back then (this code is from 2019!) it looked pretty similar:

https://github.com/kubevirt/kubevirt/blob/9bc8ff3e78ab032f1ac76d950d9c0c58a9cab0c7/pkg/virtctl/vm/vm.go#L115

My guess: Maybe the client wasn't as advanced back then or the API works differently today.

https://github.com/kubevirt/kubevirt/blob/9bc8ff3e78ab032f1ac76d950d9c0c58a9cab0c7/staging/src/kubevirt.io/client-go/kubecli/vm.go#L144

vs

https://github.com/kubevirt/kubevirt/blob/a3a9e83da6fff2de37b7fd87efb43466cd6e07d4/staging/src/kubevirt.io/client-go/generated/kubevirt/clientset/versioned/typed/core/v1/virtualmachine_expansion.go#L85

Reply:
I am guessing that this is an attempt to test that the new virtctl was not changed in a way that breaks with the new operator. If this is indeed true, I think that the PR should explain why it's not needed, or already tested better in a virtctl-specific test.

Reply:
As far as I know, we do not provide this kind of guarantee for `virtctl`. @xpivarc please keep me honest here.
Indeed, we also have a check that provides a warning in case of error and client and server versions differ from each other: https://github.com/kubevirt/kubevirt/blob/a3a9e83da6fff2de37b7fd87efb43466cd6e07d4/pkg/virtctl/root.go#L131-L139

Reply:
Updated the commit message to reflect the above.